### DISCUSSION TITLE
[Feature] Support to generate only jpg or webp thumbenails and support of AVIF thumbernails ?

### DISCUSSION THREAD

**Discussion by yodatak** [immich-app/immich#4491] [Feature Request]
### The feature

It could be cool to only have one type of thumbenails because of when we have many many pictures its could save a lot of space would love to use jpg-xl too ^

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by AngelaDMerkel** [comment]
@yodatak Neither of these strike me a good option. The goal of thumbs should be wide support and easy decode (I feel). Virtually nothing support JXL now, and as much as I would love it to be the standard it makes no sense. The same can be said of AVIF. Browsers have support, but there is no progressive decode for the images and decode has to happen without acceleration on most devices which isn't ideal for battery life.

**Comment by bo0tzz** [comment]
#12945
